test/test_irods_http_api.py

```diff
@@ -206,23 +210,51 @@ def test_common_operations(self):
         # Remove the collection.
         data = {'op': 'remove', 'lpath': new_collection_path}
         r = requests.post(self.url_endpoint, headers=headers, data=data)
-        #print(r.content) # Debug
+        logging.debug(r.content)
         self.assertEqual(r.status_code, 200)
 
         # Stat the collection to show that it does not exist.
         params = {'op': 'stat', 'lpath': new_collection_path}
         r = requests.get(self.url_endpoint, headers=headers, params=params)
-        #print(r.content) # Debug
+        logging.debug(r.content)
         self.assertEqual(r.status_code, 400)
 
         stat_info = r.json()
-        self.assertEqual(stat_info['irods_response']['status_code'], -170000) # NOT_A_COLLECTION
+        self.assertEqual(stat_info['irods_response']['status_code'], irods_error_codes.NOT_A_COLLECTION)
+
+    def test_creating_a_collection_with_insufficient_permissions_results_in_an_error(self):
```

Comment:
Are we including issue numbers on test names yet?

Reply:
No decision has been made on that. Not opposed to it, but also just trying to move fast since this is pre-1.0 _(not that that is a reason to not include issue numbers)_.

Reply:
yeah, feels early for now.  if we have an issue, then a test should reference it i think.   but not for tests we're being proactive with.

Reply:
Sounds good

Reply:
Because this is still an experimental project, I don't really want to attach issue numbers to anything just yet. I'd prefer to flesh things out a bit more. However, I can also see it the other way around. Having issue numbers in test names makes it easier to find the issue.

Reply:
Marking as resolved.